vocabularies/UI.xml

```diff
@@ -1745,6 +1745,9 @@ It is NOT tied to a data value (in contrast to [DataFieldWithIntentBasedNavigati
       <ComplexType Name="DataFieldForActionGroup" BaseType="UI.DataFieldAbstract">
         <Annotation Term="Common.Experimental" />
         <Annotation Term="Core.Description" String="Collection of OData actions and intent based navigations" />
+        <Property Name="ID" Type="Edm.String" Nullable="true">
+          <Annotation Term="Core.Description" String="Unique identifier of an action group. ID should be stable, as long as the perceived semantics of the action group is unchanged." />
```

Comment:
Unique within what? Globally? Within the annotated service document?

Reply:
Unique is indeed weird in the context of annotations and metadata. The respective UI thing that's related to the annotation has to be uniquely identified. Identifier is enough here. I will adapt the description.